### DISCUSSION TITLE
How to customize the YAML linter?

### DISCUSSION THREAD

**Discussion by sgrimee** [zed-industries/zed#6530] [Feature Requests]
Editing YAML files, I get a lot of linting errors: "Wrong ordering of key ...".
I would like to find out how to
- either disable that specific linting error (I am ok with not ordered yaml)
- or configure format-on-save to re-format the yaml with the right order of keys?
thanks

**Comment by hovsater** [comment]
I don't think this can be done right now. Whenever https://github.com/zed-industries/community/issues/585 is addressed, it should be as easy as setting [yaml.keyOrdering](https://github.com/redhat-developer/yaml-language-server#language-server-settings) to `false`.

**Comment by JosephTLyons** [comment]
Just to note, that I think we did disable that linting feature related to key ordering and it should be out soon.  Ideally, we will work on a system that surfaces these options to users, but for now, we disabled it, since not everyone was using YAML for that one specific case (GitHub Workflows).
